src/browser/api/bitcoind/index.ts

```diff
@@ -32,22 +34,11 @@ export default class BitcoinDClient {
   }
 
   public async configure(options: BitcoinDConfig): Promise<void> {
-    const clientConfig: ClientConstructorOption = {}
-    if (options.rpcUsername) {
-      this.rpcUser = options.rpcUsername
-      clientConfig['username'] = this.rpcUser
-    }
-    if (options.rpcPassword) {
-      this.rpcPassword = options.rpcPassword
-      clientConfig['password'] = this.rpcPassword
-    }
     if (options.network) {
       this.network = options.network
-      clientConfig['network'] = this.network
```

Comment:
What about the network ?

Reply:
Not sure what was done with that before, but I don't need it anymore (it's only used for determining ports)

Reply:
so should it be removed completely, since only the port matters ?

Reply:
No because it's still practical to have in the interface for the user to select the network instead of having to type the port manually I think.